R/get_usa_demographics.R

```diff
@@ -0,0 +1,101 @@
+if (base::getRversion() >= "2.15.1") {
+  utils::globalVariables(c("state.regions"))
+  utils::globalVariables(c("county.regions"))
+}
+
+#' Get a handful of demographic variables on US States from the US Census Bureau as a data.frame.
+#' 
+#' The data comes from the American Community Survey (ACS). The variables are: total population, percent White 
+#' not Hispanic, Percent Black or African American not Hispanic, percent Asian not Hispanic,
+#' percent Hispanic all races, per-capita income, median rent and median age.
+#' @param endyear The end year for the survey
+#' @param span The span of the survey
+#' @references The choroplethr guide to Census data: http://www.arilamstein.com/open-source/choroplethr/mapping-us-census-data/
+#' @references A list of all ACS Surveys: http://factfinder.census.gov/faces/affhelp/jsf/pages/metadata.xhtml?lang=en&type=survey&id=survey.en.ACS_ACS
+#' @importFrom utils data
+#' @export
+#' @examples
+#' \dontrun{
+#' # get some demographic data on US states from the 2010 5-year ACS
+#' df = get_state_demographics(endyear=2010, span=5)
+#' 
+#' # a choropleth map shows the location of the values
```

Comment:
Using the Tidyverse Style Guide, multi-sentence comments should use sentence case (i.e. start with a capital letter) and also have a period. https://style.tidyverse.org/functions.html#comments